pkg/meta/models.go

```diff
@@ -353,10 +362,12 @@ func (ms Collections) Retain(keep func(string, string, *Member) bool) {
 }
 
 // OrderedKeys returns the keys in document order.
-func (m *Collection) OrderedKeys() []string {
+func (m *Collection) OrderedKeys(searchable bool) []string {
```

Comment:
This is oddly specific. Why passing the searchable flag? Why not something else?
Something like passing a function filtering the data would be  more general.

Reply:
Not exactly what you suggested but I think it is better now.